.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,31 @@
+# csswg-drafts Pull Request
+
+## Steps
+
+**Before submitting a pull request,** please make sure the following is done:
+
+  1. Fork [the repository](https://github.com/w3c/csswg-drafts) and create your branch from `master`.
```

Comment:
First step is, if there isn't an issue and it isn't a very simple fix like markup, spelling, or grammar error fixes, file an issue for discussion.

While in some cases specific wording changes are useful to review, in many cases it makes more sense to get agreement on the nature of the changes required before crafting specific wording. Either way, discussions on significant changes need to involve more than just the patch author and the reviewer, as the CSSWG [operates by consensus](http://fantasai.inkedblade.net/weblog/2011/inside-csswg/decisions) and this requires broader participation in discussions than a typical software bugfix. So getting bogged down in the review workflow over specific wording is usually not the right first step to a significant change.